bench.cpp

```diff
@@ -0,0 +1,69 @@
+#include "api/Config.h"
+#include "api/Field.h"
+#include "api/Query.h"
+#include "api/Schema.h"
+#include "code_generation/CodeGenerator.h"
+#include <string>
+
+int main(int argc, const char *argv[])
+{
+  std::string testcase = (argc > 1) ? argv[1] : "select";
+  int parallelism = (argc > 2) ? std::stoi(argv[2]) : 1;
+  long bufferSize = (argc > 3) ? std::stoi(argv[3]) : 10000000;
+  int experimentDuration = (argc > 4) ? std::stoi(argv[4]) : 10;
+  std::string path = (argc > 5) ? argv[5] : "../data-generator/test_data.bin";
+  int period = 1;
```

Comment:
the first three arguments should be exactly same as what we have in other baselines. You can add extra ones specific to Grizzly after that, like parallelism

Reply:
We don't need experimentDuration